internal/web_package/test/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ package(default_visibility = ["//visibility:public"])
 
 rollup_bundle(
     name = "bundle",
+    auto_additional_entry_points = False,
```

Comment:
in particular I think we want to have it True here - I'd like to understand if that's breaking by changing the tests to illustrate just what that breakage is

Reply:
//internal/web_package/test2:test fails because the <script> tags look different. There's a bunch of <script type="module" and nomodule in the actual output but the test is not expecting that. 
[test.log](https://github.com/bazelbuild/rules_nodejs/files/3298439/test.log)

Reply:
paired on this in person - we now understand that the golden file changes to the HTML are desirable - enabling code splitting had the side effect of enabling differential loading